exercises/leap/leap.example

```diff
@@ -1,6 +1,3 @@
-
```

Comment:
I do it out of habit because it looks less cluttered on my machine. I'll remove them from my other PRs.